Data/projectData.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Budget Buddy",
+  "id": "budget-buddy",
+  "class": "code-project-3",
+  "creators": "Jeff, Stephanie, Vic, and Danny",
+  "date": "April 8, 2016",
+  "location": "Code Fellows 201",
+  "repoUrl": "https://github.com/stefuhnee/budget-buddy",
+  "summary": "<img src=\"images/budgetbuddy.png\" class=\"budgetbuddypic\"><p>Budget Buddy is a simple application to create a budget and track a savings goal. Budget Buddy contains five pages: home page, budget page, savings page, resource page, and an about developers page. The pages are organized around a theme of a manilla folder with navigational tabs. The application user will be asked to enter three pieces of information: total amount of income to be budgeted per month; expense totals for a variety of categories; an inspiring savings goal. The application will report back an informative pie chart of the users monthly expenses as well as a savings goal with a progress bar. The developers used javascript constructor objects and methods along with Chart.js to complete the project.</p>"
+},
+
+{
+  "name": "Bus Mall",
+  "id": "bus-mall",
+  "class": "code-project-2",
+  "creators": "Jeff Russell",
+  "date": "April 1, 2016",
+  "location": "Code Fellows 201",
+  "repoUrl": "https://github.com/jefferyrayrussell/bus-mall",
+  "summary": "<img src=\"images/bussmallproject.png\" class=\"bussmallpic\"><p>A company named Bus Mall is creating a novelty merchandise catalog to be placed on buses. The company needs some significant market research to determine which products will be lucrative.  This market research app portrays the products and receives favorability data that is stored over time and returned in charts and graphs. </p>"
+},
+
+{
+  "name": "Pizza Odyssey",
+  "id": "pizza-odyssey",
+  "class": "code-project-1",
+  "creators": "Jeff Russell",
+  "date": "March 25, 2016",
+  "location": "Code Fellows 201",
+  "repoUrl": "https://github.com/jefferyrayrussell/pizza-odyssey",
+  "summary": "<img src=\"images/pizzaproject.png\" class=\"pizzapic\"><p>A local pizza store owner is expanding to six branch locations throughout the city of Seattle. She needs help in gathering and organizing the business data she requires to profitably manage her business. Specifically the owner needs to determine the necessary number of pizza delivery drivers she must staff at each location during the hours the pizza branches are open. The pizza store owner requests a separate table for each location. Each table should include: each hour of the day, the number of pizzas sold, the number of deliveries made, and the number of delivery drivers recommended. The pizza owner would also like a form to input marketing variables so that the production needs of a new location can be calculated.</p>"
+},
```

Comment:
You should copy all of this and drop it into a JSON linter.  This isn't proper JSON.  It's pretty picky.  One comma in the wrong spot and it won't work.